.gitignore

```diff
@@ -1,2 +1,3 @@
 vendor/
 build/
+.idea
```

Comment:
@daleattree ide / platform / specific os files should be in your global gitignore, not in the project repo